examples/graphql-api/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "private": true,
+  "name": "remix-app",
+  "description": "",
+  "license": "",
+  "scripts": {
+    "build": "node -r dotenv/config node_modules/.bin/remix build",
+    "dev": "node -r dotenv/config node_modules/.bin/remix dev",
```

Comment:
I think the example shouldn't come with dotenv already installed, other examples have the .env.example file but let the developer see how to use them

Reply:
I took a look at some of the [other examples](https://github.com/remix-run/remix/search?l=JSON&q=%22dotenv%22%3A&type=code) in here which have included the package. Happy to remove it though, do you have an "example" that I could use as reference?

Reply:
This one https://github.com/remix-run/remix/tree/main/examples/collected-notes has the .env.example file but it didn’t add dotenv as dependency for example.

Reply:
Thanks Sergio I've removed it and updated the setup with a link to the [Remix docs](https://github.com/remix-run/remix/pull/1754/files#diff-67c5b8c01700db6527b0438aaee80688f741b0f1bb67b2b43936230014b029f8R8), user can choose their own implemenation path that way 👍